File: JAVASCRIPT/Questões uri/1012.js
const pegarValoress = (line) => line.split(" ").map(a => +a)
const [a,b,c] = pegarValoress(lines.shift())

let triangulo=(((a*c)/2))
let roda = ((c*c)*3.14159)
let trap=(((a +b)*c)/2)
let retan=(a*b)
let mine=(b*b)

console.log(`TRIANGULO: ${triangulo.toFixed(3)}`)
console.log(`CIRCULO: ${roda.toFixed(3)}`)
console.log(`TRAPEZIO: ${trap.toFixed(3)}`)
console.log(`QUADRADO: ${mine.toFixed(3)}`)
console.log(`RETANGULO: ${retan.toFixed(3)}`)

File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1012.py
nota= input().split(" ")
a, b, c = nota

triangulo=((float(a) * float(c)/2))
roda = (float(c)* float(c))*(3.14159)
trap=(((float(a) +float(b)) *float(c))/2)
mine=(float(b) * float(b))
retan=(float(a) *float(b))

print("TRIANGULO: %0.3f"%triangulo)
print("CIRCULO: %0.3f"%roda)
print("TRAPEZIO: %0.3f"%trap)
print("QUADRADO: %0.3f"%mine)
print("RETANGULO: %0.3f"%retan)